### ISSUE TITLE
Probleme with Firmware  upgrade

### ISSUE THREAD

**Issue by GabrielC0** [jamesshao8/portapack-havoc-modified#28]
Hello, I am asking for help as I wanted to update my firmware using flash. I flashed the file then waited 3-4 minutes before restarting the device. and when I turn on the device there is a black screen. the LEDs at the top are on and I can't access the files using the USB. can you help me please

**Comment by jamesshao8** [comment]
Hi Gabriel,

I assume you are trying to flash the firmware without computer. Sometimes, this would fail if the TF card has a problem. If that happens, you need to do a DFU recovery, and flash firmware using computer.

https://hackrf.readthedocs.io/en/latest/updating_firmware.html#only-if-necessary-dfu-boot
This page explains how to do DFU recovery, but it's mainly done under Linux.

I have also sent you some program through email, that is essentialy the same but can be used under Windows. 

Even if you successfully recover the device, you won't be able to access files using USB, unless you switch to this special mode in PortaPack menu.

**Comment by GabrielC0** [comment]
Thanks all is good now, but when a turn on the device, i need to comfirm the warning after any turn on. its normal ?

**Comment by jamesshao8** [comment]
Yes, it’s normal. I added this notification in latest firmware to make sure the device is used properly.Regards,Shao在 2024年6月8日，15:13，GabrielC0 ***@***.***> 写道：﻿
Thanks all is good now, but when a turn on the device, i need to comfirm the warning after any turn on. its normal ?
